src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_validators.py

```diff
@@ -20,11 +20,44 @@ def datetime_type(string):
                 return datetime.strptime(string, form)
             except ValueError:
                 continue
-        raise ValueError("Input '{}' not valid. Valid example: 2017-02-11T23:59:59Z".format(string))
+        raise ValueError(
+            "Input '{}' not valid. Valid example: 2017-02-11T23:59:59Z".format(string))
     return datetime_type
 
 
 def validate_both_start_end_dates(namespace):
     """Validates the existence of both start and end dates in the parameter or neither"""
     if (namespace.start_date and not namespace.end_date) or (namespace.end_date and not namespace.start_date):
-        raise CLIError("usage error: Both --start-date and --end-date need to be supplied or neither.")
+        raise CLIError(
+            "usage error: Both --start-date and --end-date need to be supplied or neither.")
```

Comment:
Move to same line... similar elsewhere.

Reply:
its result of automatic format feature of Visual Studio Code

Reply:
corrected formating